### PR TITLE
Update contributors

### PR DESCRIPTION
**Pull request by renjithvk**
Update the contributors' list. It seems some of the contributor's usernames are updated and so wrong here.

### CODE REVIEW

File: src/content/configuration/stats.md [hunk 1]
```diff
@@ -7,6 +7,13 @@ contributors:
   - jungomi
   - ldrick
   - jasonblanchard
+  - TeffenEllis
+  - jbottigliero
+  - sokra
+  - renjithvk
+  - pierreneter
+  - jasonblanchard
+  - Hoten
```
Comment (dhruvdutt) [inline src/content/configuration/stats.md]: Can you please also mention the source where you found missing contributors?
I see this on GitHub. 
image

Comment (renjithvk) [reply src/content/configuration/stats.md]: Currently here only added 4 contributors list:
editing webpack js org stats md at master webpack webpack js org

I saw there have 7 contributors:
webpack js org stats md at master webpack webpack js org

That why updated the contributors' list

Comment (dhruvdutt) [reply src/content/configuration/stats.md]: I mean the total list is now of 11 people. :D

Comment (renjithvk) [reply src/content/configuration/stats.md]: @dhruvdutt I did fix that, thanks

File: src/content/configuration/stats.md [hunk 2]
```diff
@@ -2,11 +2,13 @@
...
-  - sallar
-  - jungomi
-  - ldrick
+  - TeffenEllis
+  - jbottigliero
+  - sokra
+  - renjithvk
+  - pierreneter
   - jasonblanchard
+  - Hoten Hoten
```
Comment (dhruvdutt) [inline src/content/configuration/stats.md]: ~Hoten~ Hoten

**Comment by montogeek** [comment]
Maybe some users don't want to be on that list.
@renjithvk Could you please just add your name? We can't make decisions for others.

**Comment by renjithvk** [comment]
@montogeek Okay. I've added my username only to the contributors' list.

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**